authorization_services/topics/resource-server-enable-authorization.adoc

```diff
@@ -1,7 +1,11 @@
 [[_resource_server_enable_authorization]]
 = Enabling Authorization Services
 
+<<<<<<< HEAD
 To turn your OIDC Client Application into a resource server and enable fine-grained authorization, select *Access type* *confidential* and click the *Authorization Enabled* switch to *ON* then click *Save*.
+=======
+To turn your OIDC Client Application into a resource server and enable fine-grained authorization, set *Access Type* to *confidential*, set *Authorization Enabled* to *ON*, and click *Save*.
+>>>>>>> KEYCLOAK-9341 Updating all screens to 7.4 and eliminate branding per CCS standards
```

Comment:
I went through nearly 1/2 of the images. Only found minor issue so far.

- Weird cropping in most of the images, there should be more space around the egdes (between the edges and the text/fields).

- Images with authz "policy" forms have weird the "Logic" dropdown menu. I can only see "Logic: [Po]" instead of "Logic: [Positive]". Not sure if browser-specific or UI bug.

- Some images are too narrow. The left menu items are squeezed into multiple lines, e.g. "Identity Providers" and "User Federation". It makes the menu look weird, difficult to read. There's a lot of these, not sure if it's worth the effort redoing the screenshots.
  
List of narrow images (mostly affects the left menu):

authorization_services/images/policy/create-client.png 
authorization_services/keycloak-images/getting-started/hello-world/adapter-config.png
authorization_services/keycloak-images/getting-started/hello-world/authz-settings.png
authorization_services/keycloak-images/getting-started/hello-world/enable-authz.png
authorization_services/keycloak-images/policy-evaluation-tool/policy-evaluation-tool.png
authorization_services/keycloak-images/resource-server/authz-settings.png
authorization_services/keycloak-images/resource-server/client-enable-authz.png
authorization_services/keycloak-images/resource-server/client-list.png (multiline table items)
authorization_services/rhsso-images/getting-started/hello-world/authz-settings.png
authorization_services/rhsso-images/getting-started/hello-world/create-user.png
authorization_services/rhsso-images/getting-started/hello-world/enable-authz.png
authorization_services/rhsso-images/getting-started/hello-world/reset-user-pwd.png
authorization_services/rhsso-images/policy-evaluation-tool/policy-evaluation-tool.png
authorization_services/rhsso-images/resource-server/authz-settings.png
authorization_services/rhsso-images/resource-server/client-enable-authz.png
authorization_services/rhsso-images/resource-server/client-list.png (multiline table items)

securing_apps/keycloak-images/exchange-target-client-permission-set.png (table)
securing_apps/keycloak-images/exchange-users-apply-policy.png
securing_apps/keycloak-images/exchange-users-client-policy.png
securing_apps/keycloak-images/exchange-users-permission-set.png
securing_apps/keycloak-images/exchange-users-permission-setup.png
securing_apps/rhsso-images/exchange-idp-permission-set.png
securing_apps/rhsso-images/exchange-idp-permission-setup.png
securing_apps/rhsso-images/exchange-target-client-policy.png
securing_apps/rhsso-images/exchange-users-apply-policy.png
securing_apps/rhsso-images/exchange-users-client-policy.png
securing_apps/rhsso-images/exchange-users-permission-set.png

I still have to go through the Admin Guide which is the second half of images.